wolfcrypt/src/sp_int.c

```diff
@@ -17135,18 +17135,21 @@ static int _sp_mont_red(sp_int* a, const sp_int* m, sp_int_digit mp, int ct)
     bits = sp_count_bits(m);
 
     /* Adding numbers into m->used * 2 digits - zero out unused digits. */
-    if (!ct) {
-        for (i = a->used; i < m->used * 2; i++) {
-            a->dp[i] = 0;
-        }
-    }
-    else {
+#ifndef WOLFSSL_NO_CT_OPS
+    if (ct) {
```

Comment:
Is ct unused when WOLFSSL_NO_CT_OPS is defined?

Reply:
Nope, still gets accessed in an ungated clause at line 17222.